scripts/start-dev.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * The purpose of this script is to coordinate the start up of all of the processes necessary for development
+ * in the context of Electron. This includes:
+ *   - webpack-dev-server, which watches, recompiles, and serves code run in Electron's renderer process
+ *   - webpack, which compiles and watches the code run in Electron's main process
+ *   - electron itself, which should only be started when the others are up and running
+ *
+ * This script is intended to be run in the foreground. Sending a SIGINT to this script will clean up the child
+ * processes spawned herein.
+ */
```

Comment:
Maybe add a comment with a pointer to the original, in case we find a bug here and want to keep them both correct?

Reply:
Done in f3089ea67